packages/app/obojobo-document-engine/src/scripts/oboeditor/components/code-editor.js

```diff
@@ -16,6 +16,8 @@ import APIUtil from 'obojobo-document-engine/src/scripts/viewer/util/api-util'
 import EditorUtil from 'obojobo-document-engine/src/scripts/oboeditor/util/editor-util'
 import FileToolbar from './toolbars/file-toolbar'
 import hotKeyPlugin from '../plugins/hot-key-plugin'
+import ModalUtil from 'obojobo-document-engine/src/scripts/common/util/modal-util'
+import SimpleDialog from 'obojobo-document-engine/src/scripts/common/components/modal/simple-dialog'
```

Comment:
These should be relative since we're in the obojobo-document-engine package: `../../common/components/modal/simple-dialog`

Reply:
for that matter, line 16 should be updated too